100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████| 32/32 [00:00<00:00, 536.62it/s]
100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████| 32/32 [00:00<00:00, 460.85it/s]
  0%|                                                                                                                       | 0/32 [00:00<?, ?it/s]
Total steps: 10
Warmup steps: 1
(tensor([[  101,  1188,  5286,  ...,     0,     0,     0],
        [  101,  1252,  1646,  ...,     0,     0,     0],
        [  101, 28998, 10918,  ...,     0,     0,     0],
        ...,
        [  101,  1130,   170,  ...,     0,     0,     0],
        [  101,   141, 10964,  ...,     0,     0,     0],
        [  101, 29000,  7247,  ...,     0,     0,     0]]), tensor([[1., 1., 1.,  ..., 0., 0., 0.],
        [1., 1., 1.,  ..., 0., 0., 0.],
        [1., 1., 1.,  ..., 0., 0., 0.],
        ...,
        [1., 1., 1.,  ..., 0., 0., 0.],
        [1., 1., 1.,  ..., 0., 0., 0.],
        [1., 1., 1.,  ..., 0., 0., 0.]]), tensor([ 0,  4,  0,  0,  0, 16,  3, 25,  8,  0,  0,  0,  1,  0,  0,  3]), tensor([10, 14, 16, 21, 13, 11,  1, 15,  1, 42, 29, 63,  5,  6, 18,  1]), tensor([  4,  30,   1,  36,  27,  33,  13,   1,   6,  66,  14, 102,   2,  47,
         46,  47]))
Loss = tensor(3.7109, device='cuda:0', grad_fn=<DivBackward0>)
100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████| 32/32 [00:00<00:00, 378.58it/s]
100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████| 32/32 [00:00<00:00, 521.53it/s]
  0%|                                                                                                                        | 0/2 [00:00<?, ?it/s]
Traceback (most recent call last):
  File "train_tacred_datamap.py", line 198, in <module>
    main()
  File "train_tacred_datamap.py", line 194, in main
    train(args, model, train_features, benchmarks)
  File "train_tacred_datamap.py", line 49, in train
    for index, guid in enumerate(batch[5]):
IndexError: tuple index out of range
Traceback (most recent call last):
  File "train_tacred_datamap.py", line 198, in <module>
    main()
  File "train_tacred_datamap.py", line 194, in main
    train(args, model, train_features, benchmarks)
  File "train_tacred_datamap.py", line 49, in train
    for index, guid in enumerate(batch[5]):
IndexError: tuple index out of range